yarn.config.cjs

```diff
@@ -0,0 +1,35 @@
+/** @type {import('@yarnpkg/types')} */
+const { defineConfig } = require('@yarnpkg/types');
+
+module.exports = defineConfig({
+  async constraints({ Yarn }) {
+    for (const workspace of Yarn.workspaces()) {
+      // make sure all deps to this workspace use the correct version
+      for (const dep of Yarn.dependencies({ ident: workspace.ident })) {
+        dep.update(workspace.manifest.version);
+      }
+
+      if (!workspace.cwd.startsWith('packages/')) {
+        // all example apps should be private
+        workspace.set('private', true);
+        continue;
+      }
+      workspace.set('type', 'module');
+      workspace.set('module', 'build/esm/index.js');
+      workspace.set('types', 'build/types/index.d.ts');
+      workspace.set('files', ['dist/index.*']);
+      workspace.set('scripts.build', 'tsc');
+      workspace.set(
+        'scripts.bundle',
+        '[ ${CI:-0} != 0 ] && api-extractor run --local || api-extractor run && rollup -c',
```

Comment:
@nicklasl I see you changed this. To me it looks broken? Shouldn't there be a parentheses around the api-extractor cases?

Reply:
Yeah, I just tried it, now it doesn't run rollup at all locally. Will fix.